implementations/rust/ockam/ockam_app/src/enroll/enroll_user.rs

```diff
@@ -12,13 +11,14 @@ use ockam_command::{CommandGlobalOpts, GlobalArgs};
 ///  - connects to the Auth0 service to authenticate the user of the Ockam application to retrieve a token
 ///  - connects to the Orchestrator with the retrieved token to create a project
 #[tauri::command]
-pub fn enroll_user() -> Result<()> {
+pub fn enroll_user() {
     let options = CommandGlobalOpts::new(GlobalArgs::default());
     if options.state.identities.default().is_err() {
         create_default_identity(&options);
     }
-    embedded_node(rpc, options)?;
-    Ok(())
+    if let Err(e) = embedded_node(rpc, options) {
+        println!("Error while enrolling user: {e:?}");
```

Comment:
```suggestion
        // use tracing::error
        error!("Error while enrolling user: {e:?}");
```
or
```suggestion
        eprintln!("Error while enrolling user: {e:?}");
```

Reply:
At the time I wrote this I wasn't sure about how we were going to handle logs. Now that Eric has enabled the tauri logging plugin, makes total sense to replace all the prints to tracing logs.

Thanks for the heads up!